corehq/apps/app_manager/static/app_manager/js/apps_base.js

```diff
@@ -19,4 +19,16 @@ $(function () {
     if (v2) {
         hqLayout.utils.setIsAppbuilderResizing(true);
     }
+
+    $(document).ajaxComplete(function(e, xhr, options) {
+        if (/edit_form_attr/.test(options.url) ||
```

Comment:
Yes, this sounds like a very good idea. I was following the lead from app preview, mainly.